dispatcher_plugin_nb2workflow/dataserver_dispatcher.py

```diff
@@ -208,16 +207,19 @@ def run_query(self,
         if res.status_code == 200:
             resroot = res.json()['data'] if run_asynch else res.json()
             
-            except_message = None
-            if resroot['exceptions']: 
+            if resroot['exceptions']:
                 if isinstance(resroot['exceptions'][0], dict): # in async
                     except_message = resroot['exceptions'][0]['ename']+': '+res.json()['data']['exceptions'][0]['evalue']
                 else:
                     except_message = res.json()['exceptions'][0]
-                                                            
-                query_out.set_failed('Backend exception', 
-                                    message='Backend failed. ' + except_message,
-                                    job_status='failed')
+
+                if resroot['exceptions'][0]['ename'] == 'AnalysisError':
+                    query_out.set_status(1, message='Error in the backend', debug_message=except_message, job_status='failed')
+                else:
+                    query_out.set_failed('Analysis error',
+                                     message='Analysis error. ' + except_message,
```

Comment:
Maybe better "Backend failed" here, like before.

Reply:
error in the copy/past